WBooks/DetailBook/DetailBookViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  DetailBookViewController.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 25/08/21.
+//
+
+import UIKit
+
+class DetailBookViewController: UIViewController {
+    
+    private lazy var detailBookView = DetailBookView()
+    private let detailBookViewModel: DetailBookViewModel
+
+    init(detailBookViewModel: DetailBookViewModel) {
+        self.detailBookViewModel = detailBookViewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = NSLocalizedString("TITLE_DETAILBOOK", comment: "")
+        detailBookView.tableView.delegate = self
+        detailBookView.tableView.dataSource = self
+        detailBookView.bookImage.imageFromUrl(urlString: detailBookViewModel.book.image)
+        detailBookView.titleLabel.text = detailBookViewModel.book.title
+        detailBookView.authorLabel.text = detailBookViewModel.book.author
+        detailBookView.yearLabel.text = detailBookViewModel.book.year
+        detailBookView.genreLabel.text = detailBookViewModel.book.genre
+        detailBookView.statusLabel.text = detailBookViewModel.book.status
+        
+        detailBookView.rentButton.addTarget(self, action: #selector(rentPress), for: .touchUpInside)
+        detailBookView.addWishlistButton.addTarget(self, action: #selector(addWishlistPress), for: .touchUpInside)
```

Comment:
Split this in a method like `setupActions`

Reply:
Done